src/qibocal/protocols/characterization/allxy/allxy.py

```diff
@@ -20,6 +20,10 @@ class AllXYParameters(Parameters):
     """Beta parameter for drag pulse."""
     qubits: Optional[list] = field(default_factory=list)
     """Local qubits (optional)."""
+    update: Optional[
+        bool
+    ] = False  # False because this routine does not update the runcard
+    """Runcard update mechanism."""
```

Comment:
Has this formatting a meaning here?:)
```suggestion
    update: Optional[bool] = False  # False because this routine does not update the runcard.
    """Runcard update mechanism."""
```

Reply:
This is not what I meant, the is only the formatting, the two line breaks, I know to little about data class init functionality, that's why I asked.

Nevertheless, the idea that `False` has to be passed is not bad, meaning that all parameter objects of a routine which don't update anything have to have this attribute and it always have to be `False` and when setting it to `True` it raises an error? This is very transparent, and you can see very fast if a routine can update something or not. I like it.

Reply:
The formatting was done by pre-commit I think.
Actually I would say that this is quite strong 

> Perhaps a proper solution would be to raise an error if it is passed `update=True`

maybe let's see how this first version of `update` behave.